client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -16,7 +16,8 @@ import {
 	getImageEditorTransform,
 	getImageEditorFileInfo,
 	getImageEditorCrop,
-	isImageEditorImageLoaded
+	isImageEditorImageLoaded,
+	getImageMeetsMinimumDimensions
```

Comment:
h/t @dmsnell ^^. Redux dev tools are also highly recommended if you aren't using it yet.